src/styles/_catalog.scss

```diff
@@ -1,22 +1,32 @@
+$cardWidth: 200px;
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  align-items: center;
+  gap: 48px;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, $cardWidth);
+  }
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, $cardWidth);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $cardWidth);
+  }
 
   &__card {
-    display: flex;
+    display: grid;
     justify-content: center;
     align-items: center;
-    width: 200px;
+    width: $cardWidth;
     height: 300px;
-    border: 1px solid #ccc;
-    border-radius: 5px;
     margin-bottom: 48px;
```

Comment:
I think, you don't need this. You have the gap between your cards, and margin gives you extra space between first and second rows.